pkg/kubelet/kuberuntime/kuberuntime_manager_test.go

```diff
@@ -1659,7 +1673,7 @@ func TestComputePodActionsWithRestartableInitContainers(t *testing.T) {
 			actions: podActions{
 				SandboxID:             baseStatus.SandboxStatuses[0].Id,
 				InitContainersToStart: []int{2},
-				ContainersToKill:      getKillMapWithInitContainers(basePod, baseStatus, []int{2}),
+				ContainersToKill:      modifyKillMapContainerImage(getKillMapWithInitContainers(basePod, baseStatus, []int{2}), baseStatus, []int{2}, []string{"foo-image"}),
```

Comment:
this is a bit of a hack, but otherwise the test fails because the image name in containersToKill doesn't match